structure/stack/doubly_linked_list.go

```diff
@@ -0,0 +1,93 @@
+package stack
+
+import (
+	"container/list"
```

Comment:
Please use our internal data structures when creating new ones. Using `container` package hides a lot of material that could be used for learning.

Reply:
This code was already present (I just renamed the file). Since there is another implementation using linked list, shall I remove this one?

Reply:
Keep it there, but create a new Issue to fix this 🙏🏼